components/content-service/pkg/git/git.go

```diff
@@ -300,7 +300,7 @@ func (c *Client) Status(ctx context.Context) (res *Status, err error) {
 	}
 
 	latestCommit := ""
-	gitout, err = c.GitWithOutput(ctx, "log", "--pretty=%H", "-n", "1")
+	gitout, err = c.GitWithOutput(ctx, "log", "--pretty=%H", "--branches", "-n", "1")
 	if err != nil && !strings.Contains(err.Error(), errNoCommitsYet) {
```

Comment:
Note, I leave the `!strings.Contains(err.Error(), errNoCommitsYet)` as is to prevent other cases we missed.

Reply:
Perhaps an edge case, but this would return the latest commit in the repository right? Not the last commit from the current branch.

Reply:
But this changes behavior of git log? 

For example:
```
pavel@pavel-gitpod:~/code/gitpod-io/gitpod:[none](main)$ git log --branches -n 1
commit 37f6d53721a60b26d2eaa18abe436482f7a48896 (origin/aledbf/pvc, aledbf/pvc)
Author: Manuel Alejandro de Brito Fontes <aledbf@gmail.com>
Date:   Wed Aug 3 14:53:58 2022 -0400

    [ws-manager] Disable PVC feature
```

old:
```
pavel@pavel-gitpod:~/code/gitpod-io/gitpod:[none](main)$ git log -n 1
commit 3783204e6471b6d2e3ef3744060d68efb6c0069e (HEAD -> main, origin/main, origin/HEAD)
Author: Laurie T. Malau <laurie@gitpod.io>
Date:   Tue Aug 2 16:20:53 2022 +0000

    Replace with sorting icon
```

Basically instead of returning latest commit on main (my current branch), when you add `--branches` it returns last(?) branch?

This seems like something that we would not want to do here?